identity-account/src/account/account.rs

```diff
@@ -307,6 +313,59 @@ where
     Ok(())
   }
 
+  /// Encrypts the given `plaintext` with the specified `encryption_options`.
+  ///
+  /// Diffie-Helman key exchange with Concatenation Key Derivation Function will be performed to obtain the encryption
+  /// secret.
+  /// 
+  /// Returns the [`EncryptedData`] and the ephemeral `PublicKey` used when generating the shared secret.
+  #[cfg(feature = "encryption-decryption")]
+  pub async fn encrypt_data(
+    &self,
+    plaintext: &[u8],
+    associated_data: &[u8],
+    encryption_options: &EncryptionOptions,
+    public_key: PublicKey,
+  ) -> Result<(EncryptedData, PublicKey)> {
+    self
+      .storage()
+      .encrypt_data(
+        self.did(),
+        plaintext.to_vec(),
+        associated_data.to_vec(),
+        encryption_options,
+        public_key,
+      )
+      .await
+      .map_err(Into::into)
+  }
+
+  /// Decrypts the given `data` with the specified `encryption_options`.
+  ///
+  /// Diffie-Helman key exchange with Concatenation Key Derivation Function will be performed to obtain the decryption
```

Comment:
```suggestion
  /// Diffie-Hellman key exchange with Concatenation Key Derivation Function will be performed to obtain the decryption
```

Probably best to do a search-replace for "Helman", this affects Wasm and NAPI as well.